eng/pipelines/devops-create-package-workitem.yml

```diff
@@ -0,0 +1,68 @@
+pool:
+  name: 'azsdk-pool-mms-ubuntu-2204-general'
+  vmImage: 'ubuntu-22.04'
+
+parameters:
+- name: Language
+  type: string
+  default: ''
+  values:
+  - .NET
+  - Java
+  - Python
+  - JavaScript
+  - Go
+  - C++
+  - C
+- name: ServiceName
+  displayName: Service Name (Value should be same as service name in Service Tree)
+  type: string
+  default: ''
+- name: PackageDisplayName
+  displayName: Package Display Name ( for e.g. Azure Core)
+  type: string
+  default: ''
+- name: PackageType
+  type: string
+  default: ''
+  values:
+  - Client
+  - mgmt
+- name: PackageName
+  displayName: Package Name (e.g. Azure.Core)
+  type: string
+  default: ''
+- name: PackageVersion
+  displayName: Package Version (e.g. 1.0.0)
+  type: string
+  default: ''
+- name: ReleaseDate
+  displayName: Release Date (Format: MM/DD/YYYY)
+  type: string
+  default: ''
+- name: RelatedWorkItemId
+  displayName: Related Work Item ID (Applicable to add release plan work Item as related)
+  type: string
+  default: ''
+
+variables:
+  AzureSDKToolsScriptsPath: $(System.DefaultWorkingDirectory)/azure-sdk-tools/eng/common/scripts
+
+steps:
+- checkout: self
+
+- task: PowerShell@2
+  displayName: Create Package Work Item
+  inputs:
+    pwsh: true
+    filePath: $(AzureSDKToolsScriptsPath)/Update-DevOps-Release-WorkItem
+    arguments: >
+      -language '$(Language)'
```

Comment:
Don't you need to access these via `{ parameters.Language }}`

Reply:
You are right. I think I used variable way of accessing them. thanks